Gemfile

```diff
@@ -240,6 +240,8 @@ gem 'aws-sdk-glue'
 gem 'aws-sdk-rds'
 gem 'aws-sdk-route53'
 gem 'aws-sdk-s3'
+gem 'aws-sdk-sagemaker'
```

Comment:
My bad - I had this in my branch, but don't know if we actually need the `aws-sagemaker` gem? I think we may only need `sagemakerruntime`